pkg/controller/infrastructure/actuator_reconcile.go

```diff
@@ -395,6 +407,18 @@ func generateTerraformInfraConfig(ctx context.Context, infrastructure *extension
 		dhcpDomainName = fmt.Sprintf("%s.compute.internal", infrastructure.Spec.Region)
 	}
 
+	isIPv4 := true
+	isIPv6 := false
+	if sets.New[v1beta1.IPFamily](ipFamilies...).Has(v1beta1.IPFamilyIPv6) {
```

Comment:
Is there a special reason why we use sets, couldn't we just use `slices.Contains()` here?

Reply:
No special reason, I changed it to slices.Contains().